mail_template_multi_report/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © 2016 Savoir-faire Linux
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Mail Template Multi Report',
+    'version': '10.0.1.0.0',
+    'category': 'Marketing',
+    'summary': 'Multiple Reports in Mail Templates',
+    'author': 'Savoir-faire Linux,Odoo Community Association (OCA)',
+    'website': 'http://www.savoirfairelinux.com',
```

Comment:
Please change to the new standard https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L9